stream_alert/alert_processor/outputs/aws.py

```diff
@@ -214,16 +214,14 @@ def dispatch(self, alert, descriptor):
         # with `Qualifier=''` or `Qualifier=None` does not work and thus we
         # have to perform different calls to client.invoke().
         if qualifier:
-            resp = client.invoke(FunctionName=function,
+            return client.invoke(FunctionName=function,
                                  InvocationType='Event',
                                  Payload=alert_string,
                                  Qualifier=qualifier)
-        else:
-            resp = client.invoke(FunctionName=function,
-                                 InvocationType='Event',
-                                 Payload=alert_string)
 
-        return self._log_status(resp, descriptor)
+        return client.invoke(FunctionName=function,
+                             InvocationType='Event',
+                             Payload=alert_string)
```

Comment:
I thought `_dispatch` was supposed to return a bool? I know this (dict return) is the same functionality we had before, but I'd like to keep consistent return types (it happens in other outputs, too). Maybe we just return True at the end of this? Returning the dict doesn't make much sense anyway